models/edr/alerts/failed_tests.sql

```diff
@@ -0,0 +1,28 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+{% set anomaly_detection_relation = adapter.get_relation(this.database, this.schema, 'alerts_anomaly_detection') %}
+// Backwards compatibility support for a renamed model.
+{% set data_monitoring_relation = adapter.get_relation(this.database, this.schema, 'alerts_data_monitoring') %}
+
+with failed_tests as (
+     select * from {{ ref('elementary', 'alerts_schema_changes') }}
+     union all
+     select * from {{ ref('elementary', 'alerts_dbt_tests') }}
+
+    {% if anomaly_detection_relation %}
```

Comment:
Let's check if the relation_exists to double check we can query it and it's not cached or anything like that?

Reply:
It [seems](https://github.com/dbt-labs/dbt-core/blob/8442fb66a591c4f5353f8baaaaf62f846c5f5171/core/dbt/include/global_project/macros/adapters/relation.sql#L70) that `load_relation()` is just a user-friendly way of invoking `get_relation()`.  
Or in their words _"A convenience wrapper for get_relation."_
Could you please elaborate a bit on why do it?

Reply:
Sounds good, resolving